packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -40,6 +40,7 @@ import {
 } from '@looker/sdk-codegen'
 import { Looker31SDK, Looker40SDK } from '@looker/sdk'
 import ApiExplorer from '@looker/api-explorer/lib/ApiExplorer'
+import './styles.css'
```

Comment:
What do you think about this? Tried adding to ApiExplorer but the .css wasn't getting bundled.

Reply:
What is `limit`?

Reply:
No, the gap between `body` and `limit` isn't new, here's the current build:
![image](https://user-images.githubusercontent.com/56001784/114126192-d2997000-98ac-11eb-820b-a8df15d9dda2.png)

The request body and sdk declaration AceEditor have the default height of 500px with dynamic content size whereas the response body height is 85% of page height which we can consistently expect

Reply:
Couple of ways to solve this problem without needing an additional CSS file (and the webpack fun involved with loading that).

Specify `100vh` (100% of view-height) on the outer-most DOM element of the application layout.

Alternatively, you can use Styled Components' `createGlobalStyle` function:

```
export const BodyOverride = createGlobalStyle`
  html {
    height: 100%;
    overflow: hidden;
  }
`
```